facility-manager.html

```diff
@@ -120,12 +121,15 @@
     like to any user's data, in any group (except annotate in Private groups)
     and Group Owners can do anything to any user's data in the groups they
     own.
+    <a href="https://docs.openmicroscopy.org/omero/5.3.3/sysadmins/admins-with-restricted-privileges.html" class="external-url">
+    Restricted Administrators</a> were introduced in OMERO 5.4.0 and can
```

Comment:
``Restricted Administrators level was...``

Reply:
Restricted Administrators is not really a single level. I'll fix the previous sentence and use "roles" instead of "levels".